runtime/shibuya/src/lib.rs

```diff
@@ -321,7 +325,7 @@ impl frame_system::Config for Runtime {
     type OnNewAccount = ();
     type OnKilledAccount = pallet_unified_accounts::KillAccountMapping<Self>;
     type DbWeight = RocksDbWeight;
-    type BaseCallFilter = BaseFilter;
+    type BaseCallFilter = BaseCallFilter;
```

Comment:
Does this require the adjustment of base extrinsic weight?

There is one additional DB read for the `TxPause` map, and (negligible cost) logic for the `SafeMode` pallet.